Frontend app build failed

Failed to build source code of `frontend` app with [asc-deploy.sh](https://github.com/Azure-Samples/animal-rescue/blob/Azure/scripts/asc_deploy.sh) script, there are missing package errors in build logs, causing build failed. Below is the build error:

```
5 of 16 buildpacks participating
paketo-buildpacks/ca-certificates 2.4.1
tanzu-buildpacks/node-engine      0.6.0
tanzu-buildpacks/npm-install      0.3.0
tanzu-buildpacks/node-module-bom  0.1.1
tanzu-buildpacks/npm-start        0.2.0
...
  | | | | |   npm ERR! missing: axios@0.26.1, required by frontend@1.0.0
    npm ERR! missing: prop-types@15.8.1, required by frontend@1.0.0
    npm ERR! missing: react-responsive-carousel@3.2.23, required by frontend@1.0.0
    npm ERR! missing: wait-on@5.3.0, required by frontend@1.0.0
    npm ERR! missing: follow-redirects@1.14.9, required by axios@0.26.1
    npm ERR! missing: prop-types@15.8.1, required by react@16.14.0
    npm ERR! missing: prop-types@15.8.1, required by react-dom@16.14.0
    npm ERR! missing: classnames@2.3.1, required by react-responsive-carousel@3.2.23
    npm ERR! missing: prop-types@15.8.1, required by react-responsive-carousel@3.2.23
    npm ERR! missing: prop-types@15.8.1, required by react-easy-swipe@0.0.21
    npm ERR! missing: @typescript-eslint/eslint-plugin@4.33.0, required by react-scripts@4.0.3
    ...
    npm ERR! missing: @hapi/hoek@9.2.1, required by @hapi/topo@5.1.0
    npm ERR! missing: follow-redirects@1.14.9, required by axios@0.21.4
    +-- UNMET DEPENDENCY http-proxy-agent@4.0.1
```

--- a/README.md
+++ b/README.md
@@ -351,6 +351,9 @@ az spring-cloud app deploy --name $FRONTEND_APP \
     --source-path frontend/
 ```
 
+> Note:
+> if you've run `npm install` before deploying, remove the directory with `rm -rf frontend/node_modules` before deploying. 
+
 ### Access the Application through Spring Cloud Gateway
 
 Retrieve the URL for Spring Cloud Gateway and open it in a browser:

--- a/scripts/asc_deploy.sh
+++ b/scripts/asc_deploy.sh
@@ -79,6 +79,7 @@ function deploy_backend() {
 function deploy_frontend() {
   echo "Deploying frontend application"
   pushd $PROJECT_ROOT/frontend
+  rm -rf node_modules/
   az spring-cloud app deploy --name $FRONTEND_APP --builder nodejs-only --source-path .
 
   pushd $PROJECT_ROOT

